orchagent/vxlanorch.cpp

```diff
@@ -684,6 +1364,11 @@ bool VxlanTunnelOrch::addOperation(const Request& request)
     SWSS_LOG_ENTER();
 
     auto src_ip = request.getAttrIP("src_ip");
+    if (!src_ip.isV4())
```

Comment:
IPv6 tunnel is supported for Vnet feature. (V4 over V6 tunnels). Please remove this check